### PR TITLE
RCPS-32 web-frontend-maintenance: Refactor Guest Violation Module

### PR DESCRIPTION
**Pull request by HaroldDinglasan**

### CODE REVIEW

File: src/pages/Login.jsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ const Login = () => {
                         sessionStorage.setItem('role', authorities[2]);
                         navigate('/admin/offense');
                     } else if (authorities[1] === "ROLE_ROLE_GUEST") {
+                        alert(response.data);
```
Comment (zhairanicolepacheco) [inline src/pages/Login.jsx]: Remove unnecessary alert

**Review by zhairanicolepacheco — CHANGES_REQUESTED** [review]

**Review by JustineDaveMagnaye — APPROVED** [review]

**Review by zhairanicolepacheco — APPROVED** [review]

File: src/Constants.js [hunk 2]
```diff
@@ -1,32 +1,46 @@
 const prod = {
```
Comment (esdi-00) [inline src/Constants.js:1]: This file is getting ridiculously large and unmaintainable. In the next commit, just use one url each for dev and prod then just concatenate other parts of the api url when needed

File: src/component/AddViolationModal.jsx [hunk 3]
```diff
@@ -71,7 +71,7 @@ const AddViolationModal = ({ isOpen, onClose, onSubmit }) => {
     useEffect(() => {
         const fetchData = async () => {
             try {
-                const token = localStorage.getItem('token');
+                const token = sessionStorage.getItem('token');
```
Comment (esdi-00) [inline src/component/AddViolationModal.jsx]: why use a session storage? jwt is local storage based

**Review by esdi-00 — CHANGES_REQUESTED** [review]
address the local storage vs session storage issue

**Review by PhilippaulDaganio — APPROVED** [review]

**Review by zhairanicolepacheco — APPROVED** [review]

**Pull request closed without merging**